README.md

```diff
@@ -1,24 +1,142 @@
-# README
+# budget-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#[project])
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Available Scripts](#available-scripts)
+- [👥 Authors](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgments)
+- [📝 License](#license)
 
-* Ruby version
+# Budget App
 
-* System dependencies
+**Budget App** is a web mobile friendly app build using Ruby on Rails framework. This App supports user registration and authentication using the Devise gem. In this App, a user can create his own account on the app and after logging in she/he will be able to add categories and then add transactions under those categories. The app provide some metrics like the total value of transactions under each category.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML, CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features
 
-* ...
+- **Register an account**
+- **Create categories and transactions**
+- **Metrics about user transactions**
+
+## Live Demo
+
+- [Live Demo Link](https://my-budget-c4p6.onrender.com/)
+- [Video Link](https://drive.google.com/file/d/1dScLkKVJfj_PRMRRbhEUyU1AhxLqyC-3/view?usp=sharing)
```

Comment:
# Required Changes ♻️

- [ ] To have a great presentation that will leave a good impression about your work to a potential employer, it would be nice to have our presentation recorded in a noise-free background or an echo-free room. Let's try not to have a disturbing video presentation of our project as they help sell us out to potential employers or sell your project and presentation skills out to potential viewers of your project. Kindly redo the video in a quiet and disturbing-free background 👍🏽
- [ ] Please do some live demos of the features of your app as required in the video presentation requirement so viewers see that what you entered in the input fields really persisted. 👍🏽
- [ ] In your presentation, you forgot to show a piece of code you're proud of implementing. It's required that we show parts of our code we're proud of implementing. Kindly do that on your next video 👍🏽

**The Live Demo Version**

- [ ] The app is not working online as expected. I keep getting this screen whenever I try to signup or Login. Please check it out so it works fine online as expected 👍🏽

<img width="458" alt="Screenshot 2023-07-15 at 7 02 42 PM" src="https://github.com/brainskev/Budget_App/assets/50941074/e32e19a6-43e2-44ba-a14f-c840c040ac44">